src/topotoolbox/graphflood.py

```diff
@@ -1,42 +1,82 @@
 """
-Basic interface to libtopotoolbox implementation of graphflood
-
+Basic interface to libtopotoolbox implementation of graphflood.
 """
 
-from .grid_object import GridObject
-import numpy as np
-from ._graphflood import (  # type: ignore
-    graphflood_run_full,
-    graphflood_sfgraph,
-    compute_priority_flood_plus_topological_ordering,
-    compute_priority_flood,
-    compute_drainage_area_single_flow
-    )
 from copy import deepcopy
 
+import numpy as np
+
+# pylint: disable=no-name-in-module
+from . import _graphflood  # type:ignore
+from .grid_object import GridObject
+
 # exposing function as string dictionary
 funcdict = {
-    "run_full": graphflood_run_full,
-    "sfgraph": graphflood_sfgraph,
-    "priority_flood_TO": compute_priority_flood_plus_topological_ordering,
-    "priority_flood": compute_priority_flood,
-    "drainage_area_single_flow": compute_drainage_area_single_flow,
+    "run_full": _graphflood.graphflood_run_full,
+    "sfgraph": _graphflood.graphflood_sfgraph,
+    "priority_flood_TO":
+    _graphflood.compute_priority_flood_plus_topological_ordering,
+    "priority_flood": _graphflood.compute_priority_flood,
+    "drainage_area_single_flow": _graphflood.compute_drainage_area_single_flow,
 }
 
 __all__ = ["run_graphflood"]
 
 
 def run_graphflood(
     grid: GridObject,
-    initial_hw=None,
-    BCs=None,
-    dt=1e-3,
-    P=10 * 1e-3 / 3600,
-    manning=0.033,
-    SFD=False,
-    D8=True,
-    N_iterations=100,
+    initial_hw: np.ndarray | GridObject | None = None,
+    BCs: np.ndarray | GridObject | None = None,
+    dt: float = 1e-3,
+    P: float | np.ndarray | GridObject = 10 * 1e-3 / 3600,
+    manning: float | np.ndarray | GridObject = 0.033,
+    SFD: bool = False,
+    D8: bool = True,
+    N_iterations: int = 100,
```

Comment:
The linting is failing because of these var names. I could change this to all lower case for now until we decide on something different (or keep it the same but all lowercase).